standalone_tests/artifact_object_reference_test.py

```diff
@@ -389,6 +386,7 @@ def test_get_artifact_obj_by_name():
         assert actual_table.columns == columns
         assert actual_table.data[0][columns.index("Image")] == image
         assert actual_table.data[1][columns.index("Image")] == _make_wandb_image("2")
+        actual_table._eq_debug(_make_wandb_table(), True)
```

Comment:
Are these debug calls still supposed to be here? I thought they were for dropping in temporarily on a misbehaving test.

Reply:
yes, since they will give a better error description.

Reply:
Got it. What about the `==` below, then -- is that redundant?

Reply:
well i figured there is a non zero chance something get borked in implementation between _eq_debug and __eq__ so i kept both for good measure.